### ISSUE TITLE
quieter output in Jupyter notebooks

### ISSUE THREAD

**Issue by stevengj** [NanoComp/mpb#116]
By default, MPB prints a lot of information — this stems from its original mode of operation, where you typically piped all of the output to a file and `grep`-ed for the data you want.   When you run it in a Jupyter notebook however, and are obtaining the data programmatically rather than by grepping, all of this output is annoying.

It would be good to implement a "quiet" mode (i.e. `verbosity` scale similar to Meep), and perhaps turn it on by default during interactive use, to suppress most of this output.

Basically, this means:

1. Adding a `verbosity` global variable to MPB (I think we already have a `verbose` global Boolean, but it would be better to have an integer similar to Meep so that 0=minimal output, 1=a little, and 2=a lot=current output, 3=debugging output=old `verbose` mode)

2. Guard more of the MPB output with verbosity checks so that, e.g. frequencies aren't outputted in `verbosity<2` mode.

3. Default to `verbosity=1` mode for interactive Python usage like notebooks.

**Comment by stevengj** [comment]
verbose=1 should suppress all of the `tefreqs:, X, Y, Z` etcetera output of CSV data.
